### ISSUE TITLE
After restarting the app login is blocked

### ISSUE THREAD

**Issue by matthiasbalke** [guiqui/k8Studio#2]
When I login into the app, close it and restart it I cannot login anymore.

**Error message:**
"There is a session already open for this user"

**Workaround:**
Wait one minute or so.

That brings me to a question, what is the session used for? I get that you might want to protect your beta from public access, but if a public sign up is enabled, and it's a locally installed product there is no need for a login at all.

**Comment by guiqui** [comment]
Yes, it is. Is a way of protecting that that is open many times with the same login.
